src/transformStateWithClones.js

```diff
@@ -7,7 +7,38 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const clone = Object.assign({}, state);
+  const out = [];
```

Comment:
Same with out, what does it mean? Let's try to use more descriptive names
![image](https://user-images.githubusercontent.com/96409563/223953997-51f2146e-3a0a-4e87-84ff-ab8f01cb8c3f.png)